rooms.js

```diff
@@ -1067,6 +1067,16 @@ class BasicChatRoom extends BasicRoom {
 		this.log.modlog(message);
 		return this;
 	}
+	/**
+	 * @param {string[]} userids
+	 */
+	hideText(userids) {
+		this.log.clearText(userids);
+		for (const userid of userids) {
+			this.add(`|unlink|hide|${userid}`);
```

Comment:
`send`, not `add`.

Also maybe `clearText` could return the list of userids actually hidden, so you don't iterate through an entire alt list uselessly.

Reply:
unlink works globally though, so there might be a case somewhere for keeping it in the log